src/styles/main.scss

```diff
@@ -1,3 +0,0 @@
-body {
```

Comment:
Bom dia, por favor utiliza o main.scss como file de entrada dos códigos, para os testes é necessário este arquivo para leitura. Está no descritivo da atividade:

`Start writing styles in the main.scss file. It is be explained in the Sass lesson`